app/pages/admin/project-status.jsx

```diff
@@ -241,20 +265,44 @@ class ProjectStatus extends Component {
                   />
                   no max height
                 </label>
-              </fieldset>
-              <label>
-                <input
-                  id="grouped"
-                  type="checkbox"
-                  onChange={event => this.toggleWorkflowGrouped(event, workflow)}
-                  defaultChecked={workflow.grouped}
-                />
+              </div>
+              <hr />
+              <div>
+                <h4>Subject selection settings</h4>
+                <label>
+                  <input
+                    id="grouped"
+                    type="checkbox"
+                    onChange={event => this.toggleWorkflowGrouped(event, workflow)}
+                    defaultChecked={workflow.grouped}
+                  />
                   Use grouped subject selection
-              </label>
-            </li>
+                </label>
+              </div>
+              <hr />
+              <div>
+                <h4>Classifier 2.0 (rewrite) settings</h4>
+                <label>
+                  Subject Viewer:{' '}
+                  <select
+                    id="subject-viewers"
+                    onChange={(event) => this.onChangeSubjectViewer(workflow, event)}
+                    value={workflow.configuration.subject_viewer}
+                  >
+                    <option value="none">None Selected</option>
+                    <option value="dataImage">Data Image</option>
+                    <option value="lightCurve">(D3/TESS) Light Curve</option>
```

Comment:
This is `lightcurve` in the monorepo code but maybe we should change it there?
https://github.com/zooniverse/front-end-monorepo/blob/80ce4f31ce3ab3c1f5efd24c3f71e0d70f755198/packages/lib-classifier/src/store/Workflow/WorkflowConfiguration/WorkflowConfiguration.js#L8-L15

Reply:
It's also `lightCurve`: https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/SubjectViewer/helpers/getViewer/getViewer.js

This is the file I used as reference, so presumably it's working as `lightCurve` or else the viewer component itself wouldn't render? I'm not sure.

Reply:
Weird. Maybe the workflow config model is wrong?

Reply:
I did a bit of a digging and `workflow.configuration.viewerType` is used to set `subject.viewer`. I'm not sure what the difference is between that and the `getViewer` helper function.
https://github.com/zooniverse/front-end-monorepo/blob/80ce4f31ce3ab3c1f5efd24c3f71e0d70f755198/packages/lib-classifier/src/store/Subject/Subject.js#L81-L109

Reply:
Oh, this is very convoluted. The workflow config uses the enumeration from `@helpers/subjectViewers`, which translates `lightcurve` to `lightCurve`.
https://github.com/zooniverse/front-end-monorepo/blob/80ce4f31ce3ab3c1f5efd24c3f71e0d70f755198/packages/lib-classifier/src/helpers/subjectViewers/subjectViewers.js#L8-L11

So, if I'm reading the code correctly, the workflow config stored in Panoptes is `lightcurve`, which the classifier converts internally to `lightCurve`, which is then used to lookup the viewer component in `getViewer`.

Reply:
Thanks for looking. That is really convoluted. I think I'll update this here to use `lightcurve`, then also update `lightCurve` to be `lightcurve` where applicable in the code there so it's all consistent. The inconsistency makes for easy mistakes... like I almost did here.

Reply:
```suggestion
                    <option value="lightcurve">(D3/TESS) Light Curve</option>
```

Reply:
Sounds like a good plan. I haven't tried this in the code, but now that `workflow.configuration.subject_viewer` is an enumeration in the model, I think the `@helpers/subjectViewers` enumeration is redundant and could be removed, along with the `workflow.configuration.viewerType` view.

Reply:
Yep it's redundant, so it could be simplified.